app/src/main/java/chat/rocket/android/chatroom/adapter/MessageReactionsAdapter.kt

```diff
@@ -72,21 +72,23 @@ class MessageReactionsAdapter : RecyclerView.Adapter<RecyclerView.ViewHolder>()
     }
 
     fun contains(reactionShortname: String) =
-            reactions.firstOrNull { it.shortname ==  reactionShortname} != null
+        reactions.firstOrNull { it.shortname == reactionShortname } != null
 
     class SingleReactionViewHolder(view: View,
                                    private val listener: EmojiReactionListener?)
```

Comment:
Wrong formatting - but it was already there (it is not your "fault" in this PR).
Just an "alert" (we can fix later).

Reply:
We should agree on a formatting spec (tabs vs spaces, number of spaces, etc)